neuraxle/base.py

```diff
@@ -1527,6 +1530,12 @@ def fit(self, data_inputs, expected_outputs=None) -> 'FitCallbackStep':
         fit methods are not implemented
     """
 
+    def handle_transform(self, data_container: DataContainer, context: ExecutionContext) -> DataContainer:
+        return data_container
+
+    def handle_fit_transform(self, data_container: DataContainer, context: ExecutionContext) -> ('BaseStep', DataContainer):
```

Comment:
same comment than the previous maybe.

Reply:
only calling the wrappe methods. please, look at the new code, and reopen if needed